Sources/Apollo/GraphQLResult.swift

```diff
@@ -36,8 +36,6 @@ public struct GraphQLResult<Data: RootSelectionSet> {
   public let errors: [GraphQLError]?
   /// A dictionary which services can use however they see fit to provide additional information to clients.
   public let extensions: [String: AnyHashable]?
-  /// Metadata of this result.
-  public let metadata: GraphQLResultMetadata
```

Comment:
❗ Removing it is going to cause a lot of other down-stream breakage that is unnecessary.

Instead, let's make the initializer public and later we'll remove it when we're no longer referencing it

Reply:
ok it makes sense, I thought it made some sense adding some progress for when we move to mainline Apollo

Reply:
Yeah, I just really don't want to have another epic branch to maintain with the network layer refactor 😢